HISTORY.md

```diff
@@ -0,0 +1,7 @@
+History
```

Comment:
Nit: is `HISTORY.md` more prevalent than `CHANGELOG.md`? Maybe worth changing depending on what our big projects use...

Reply:
Yeah, I think `CHANGELOG.md` is more common for the TF modules I'm seeing. I'll change it.